apps/openssl.cnf

```diff
@@ -60,6 +60,11 @@ default = default_sect
 # included fipsmodule.cnf.
 # fips = fips_sect
 
+# XXX If no providers are activated, the default one is activated implicitly.
```

Comment:
Why the `XXX`?

Reply:
I intended to draw the vim users' attention because of syntax highlighting. I see it just draw the update checker's attention :(

Reply:
Fixed, though I'd prefer some sort of highlight to increase the chance of drawing someone's attention